changelog/unreleased/kong-ee/add-messages-api-to-anthropic.yml

```diff
@@ -0,0 +1,5 @@
+"message": "**AI-Proxy**: To support the new messages API of `Anthropic`, the upstream path of the `Anthropic` for `llm/v1/chat` route type is changed from `/v1/complete` to `/v1/messages`"
```

Comment:
would this be a breaking change to mention in changelog?

Reply:
@fffonion Let me mention it in changelog, thanks!